rdt/transformers/categorical.py

```diff
@@ -207,7 +207,7 @@ def _reverse_transform_by_matrix(self, data):
         is_data_greater_than_starts = (data >= starts)[:, ::-1]
         interval_indexes = num_categories - np.argmax(is_data_greater_than_starts, axis=1) - 1
 
-        get_category_from_index = list(self.starts['category']).__getitem__
+        get_category_from_index = list(self.starts['category']).__getitem__  # pylint:disable=E1136
```

Comment:
I have no idea why these started to fail lint. I'm assuming some library just got released which makes them fail?